opsworks_deploy_python/attributes/default.rb

```diff
@@ -8,4 +8,4 @@
 node.default["deploy_python"]["create_dirs_before_symlink"] = ['public', 'tmp']
 node.default["deploy_python"]["packages"] = ['future']
 node.default["deploy_python"]["os_packages"] = []
-node.default["deploy_python"]["venv_options"] = '--no-site-packages --python=python3.6'
```

Comment:
Can we also remove the "no site packages" flag from [the impact](https://github.com/masschallenge/opsworks-web-python/blob/master/impact_deploy/recipes/default.rb#L28) deploy as well?